build/Dockerfile.openshift

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14 AS builder
```

Comment:
Something's not right here. This is moving the golang version back to 1.19. It needs to stay 1.20.

Reply:
I will try a new rebase, let's see.